frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx

```diff
@@ -20,6 +20,8 @@ type ODFSystemDashboardPageProps = Omit<DashboardsPageProps, 'match'> & {
   match: Match<{ systemName: string }>;
 };
 
+const blockPoolRef = referenceForModel(CephBlockPoolModel);
```

Comment:
This is deprecated , use getGroupVersionKindForModel

Reply:
ack